# Gene Set Enrichment Analysis

###[Mootha et al. (2003)](https://doi.org/10.1038/ng1180), [Subramanian et al. (2005)](https://doi.org/10.1073/pnas.0506580102)

##### Title: 

##### 1. PGC-1Î±-responsive genes involved in oxidative phosphorylation are coordinately downregulated in human diabetes 
##### 2. Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles

##### Author(s): 

##### 1. Mootha et al.

##### 2. Subramanian et al.

Both papers described a method called Gene Set Enrichment Analysis (GSEA) for analyzing gene expression data. Instead of focusing on individual genes, which can be statistically unidentifiable, GSEA aims to detect modest but coordinate changes in the expression of groups of functionally related genes. The main idea of GSEA is to:

   1. Calculate Enrichment Scores (ES) for selected gene sets
   2. Estimate the significance level of ES to get p-values
   3. Adjust for multiple testing. 

It seems to me GSEA is a reinvention of the Kolmogorov-Smirnov statistic and permutation test. 

   Questions:

   1. The GSEA proposed in Subramanian et al. seems to be a weighted version of GSEA proposed in Mootha et al. Why did they say "In the original implementation, the running-sum statistic used equal weights at every step, which yielded high scores for sets clustered near the middle of the ranked list"? How to interpret Fig. 2 S3?



